packages/frontend/src/components/campaigns/index.tsx

```diff
@@ -208,11 +212,45 @@ export function Campaigns() {
     function handlePage(page: number) {
         setPageNumber(page);
     }
-
     const handleCreateCampaign = useCallback(() => {
         localizedRouter.push("/campaigns/create");
     }, [localizedRouter]);
 
+    const statusSelectRenderOption = (option: {
```

Comment:
I think we can keep these in the same file, but we should move them outside of the component (it would be a bit cleaner imo).